packages/gatsby-plugin-utils/src/polyfill-remote-file/utils/url-generator.ts

```diff
@@ -28,10 +28,12 @@ export function generateImageArgs({
   format,
   cropFocus,
   quality,
+  aspectRatio,
```

Comment:
it would also be good to pass along the `backgroundColor` via the url so that when people use fit=contain the letterboxing can be done with the appropriate colour